udata_gouvfr/tasks.py

```diff
@@ -0,0 +1,49 @@
+import mongoengine
+import requests
+
+from collections import defaultdict
+from flask import current_app
+
+from udata.commands import success, error
+from udata.core.dataset.models import Dataset
+from udata.tasks import job
+
+from udata_gouvfr import APIGOUVFR_EXTRAS_KEY
+
+
+def get_dataset(id_or_slug):
+    obj = Dataset.objects(slug=id_or_slug).first()
+    return obj or Dataset.objects.get(id=id_or_slug)
+
+
+def process_dataset(d_id, apis):
+    try:
+        dataset = get_dataset(d_id)
+    except (Dataset.DoesNotExist, mongoengine.errors.ValidationError):
+        return error(f'Dataset {d_id} not found')
+    dataset.extras[APIGOUVFR_EXTRAS_KEY] = apis
+    dataset.save()
+    success(f'Imported {len(apis)} API(s) for {str(dataset)}')
+
+
+@job('apigouvfr-load-apis')
+def apigouvfr_load_apis(self):
```

Comment:
You don't have tests for this job. I agree it's fairly simple but if you feel like writing tests 👍